dataverse-webapp/src/main/resources/config/dataverse.default.properties

```diff
@@ -64,6 +64,8 @@ ZipUploadFilesLimit=1000
 MultipleUploadFilesLimit=1000
 GuestbookResponsesPageDisplayLimit=5000
 TabularIngestSizeLimit=-1
+# Size (in cells, ie. cases times variables) of greatest table that will be ingested in-memory. Above that other method will be used.
+IngestMethodChangeThreshold=1000000
```

Comment:
That is format-agnostic parameter as it applies to every ingested file. I've done also a fix for CSV reader, but it will be delivered in another pull request.